### ISSUE TITLE
raw data has exactly one entry with empty list for "senses" field

### ISSUE THREAD

**Issue by jmviz** [tatuylonen/wiktextract#139]
In the most recent (2022-06-20) raw data (https://kaikki.org/dictionary/raw-wiktextract-data.json.gz), I noticed there is exactly one entry whose "senses" field contains only an empty list. Namely:

```json
{"word": "saying", "lang": "English", "lang_code": "en", "pos": "adj", "senses": [], "source": "thesaurus"}
```

(This entry is not in the post-processed data.)

Since there is exactly one of these entries among all 8276310, I figured this may not be intended behavior.

**Comment by kristian-clausal** [comment]
Good catch. It seems that this is caused when the parser is parsing https://en.wiktionary.org/wiki/Thesaurus:saying. The section for nouns there is parsed correctly, but the one for Adjective is not. This is a bit deeper magic, so Tatu would have to take a look at it, but definitely looks like a bug.

**Comment by kristian-clausal** [comment]
Mostly the issue is the formatting of the Wiktionary page (although I think Tatu might also take a look at other ways to parse it, as usual), the Adjectives section just needs a bit more love and more subtitles to indicate where the senses are.

**Comment by tatuylonen** [comment]
I just committed changes that relate to this.  Words coming from thesaurus entries without a sense should now have an empty sense and be marked with no-gloss.  There were also other changes related to the no-gloss, empty-gloss tags and no-sense tag was eliminated.

**Comment by jmviz** [comment]
Closing this as I believe it was resolved by bec533c6baf81f5e179990a8752d3e25061349c2.
